layout_and_module_task/app/Http/Controllers/TaskController.php

```diff
@@ -27,7 +37,7 @@ public function index()
      */
     public function create()
     {
-        $users = DB::table('users')->select('id', 'name')->get();
+        $users = $this->userRepository->getIdAndName();
```

Comment:
$this->userRepository->getUsers(['id', 'name']);

Reply:
truyền mảng là column muốn select để có thể tận dụng được function cho những chỗ khác nha Lý